reactive/archive_auth_mirror.py

```diff
@@ -1,14 +1,18 @@
 from charmhelpers.core import hookenv
 
-from charms.reactive import when, when_not
+from charms.reactive import when, when_not, hook
 
 from charms.layer.nginx import configure_site
 from charms.archive_auth_mirror import gpg, reprepro, utils
 
 
+@hook('install')
+def install():
+    utils.install_resources()
```

Comment:
As discussed, @hook is deprecated, so I moved the install handler to a @when_not one, and added an explicit installed state